index.js

```diff
@@ -128,16 +123,11 @@ function buildRouteRate (pluginComponent, params, routeOptions) {
           params.onExceeded(req)
         }
 
-        res.type('application/json').serializer(serializeError)
```

Comment:
I think we could keep this line adding a bool to know if the `errorMessage` has been overwritten by user or not.
This should restore also the changes in tests `t.strictEqual(res.headers['content-type'], 'application/json; charset=utf-8')`